tests/ui/macros/macro-rules-attr.stderr

```diff
@@ -0,0 +1,21 @@
+warning: rule #3 of macro `local_attr` is never used
+  --> $DIR/macro-rules-attr.rs:43:9
+   |
+LL |     attr() {} => {
+   |         ^^^^^
```

Comment:
Weird span.

Reply:
This span is used exclusively for unused rules, for which bang-style macros use the span of the matcher as well. An attr rule being used depends on both the arguments and the body, so this points to both.

That said, this should have used a `MultiSpan`; I just hadn't gone to the trouble to switch to one. I've now done so.